config/pacbio_smrt_link_versions.yml

```diff
@@ -36,7 +36,10 @@ default: &default
       name: v13_sequel_iie
       active: true
       default: false
-
+    v13_1_sequel_iie:
```

Comment:
We will also need a 13.1 version for Revio. Although there are no changes it is best for consistency.